style.css

```diff
@@ -1,20 +1,381 @@
+  }
+
+  .social-bar li a {
+    margin: 0 5px;
+  }
+
+  .social-bar ul {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .social-bar li {
+    list-style: none;
+  }
+
+  .touch-links h3 a {
+    color: #7f8cff;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    text-decoration: none;
+  }
+
+  .social-bar a img {
+    width: 19px;
+    height: 19px;
+  }
+
+  .social-bar h3 a {
+    padding: 0 5px;
+  }
+
+  .social-bar {
+    padding: 11px 0;
+  }
+
+  .work-section {
+    margin-top: 10px;
+    display: grid;
+    padding: 5vh 5vw;
+
+    /* border: 1px solid black; */
+    height: 100vh;
+    grid-template-rows: 35vh 10vh 10vh 76px 10vh 10vh;
+    grid-template-columns: 100%;
+    background-color: #fff;
+    gap: 5px;
+
+    /* grid-gap: 5px; */
+  }
+
+  .work-imgs {
+    background-image: url(image-resources/file.png);
+    background-size: cover;
+
+    /* border: 1px solid black; */
+  }
+
+  .work-topic {
+    font-family: 'Poppins', sans-serif;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+    padding: 10px 0;
+    color: #172b4d;
+    background-color: #fff;
+  }
+
+  .work-topic h2 {
+    background-color: #fff;
+  }
+
+  .work-details ul {
+    background-color: #fff;
+    display: flex;
+    flex-direction: row;
+    height: 5vh;
+    align-items: center;
+    list-style: none;
+  }
+
+  .work-details ul li {
+    padding: 0 5px;
+  }
+
+  .work-2nd {
+    color: #7a869a;
+  }
+
+  .work-langs ul {
+    display: flex;
+    flex-direction: row;
+    list-style: none;
+  }
+
+  .work-langs ul li {
+    background: #ebebff;
+    margin: 0 5px;
+    padding: 2px 15px;
+    border-radius: 5px;
+  }
+
+  .about-connect h3 a {
+    color: #7f8cff;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    text-decoration: none;
+  }
+
+  .work-langs ul li a {
+    text-decoration: none;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 16px;
+  }
+
+  .work-button button {
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    border: 1px solid #6070ff;
+    background-color: #2230d2;
+    border-radius: 8px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #fff;
+    width: 150px;
+    height: 7vh;
+    padding: 12px;
+    gap: 10px;
+    justify-content: center;
+  }
+
+  .about-button button {
+    box-sizing: border-box;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    background-color: #2230d2;
+    padding: 12px;
+    gap: 10px;
+    width: 154px;
+    height: 48px;
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+    line-height: 24px;
+    font-weight: 500;
+    color: #fff;
+    letter-spacing: 0.03em;
+  }
```

Comment:
- **[OPTIONAL]** the button style state is different from the Figma file, as required ensure to duplicate the design as much as possible ✨

### Your Design
![image](https://user-images.githubusercontent.com/68030297/219604231-39474ba9-9a0d-4996-8ff1-f6af472aeed4.png)


### Figma Design
![image](https://user-images.githubusercontent.com/68030297/219604335-33730f0d-a9e3-47a5-ab29-e54999fcabd0.png)